config/resources/defaults.yaml

```diff
@@ -17,6 +17,18 @@
 Debug: false
 Logging:
   Level: "Error"
+  Cache:
+    Scitokens: error
+    XrootdPssSetOptCache: error
+    Pss: error
+    Cms: error
```

Comment:
Other way around: cms is for origins, not caches

Reply:
Oh okay, why is it set within the `xrootd/resources/xrootd-cache.cfg` and not `xrootd-origin.cfg`? Was this a mistake or is this how it is supposed to work?

Reply:
I think that was a mistake. `cms` refers to the `cmsd` daemon, which is the component of xrootd that advertised the hosted paths to a central redirector so caches know where to pull data they don't have from. It needs to run on origins but it doesn't really make sense to run on caches because caches pull data from origins, not each other.